.gitattributes

```diff
@@ -3,3 +3,8 @@
 * text=auto
 
 *.sh text eol=lf
+
+# Treat all files inside .yarn/ as binary files. This ensures they do
+# not make a mess in results for git grep or git diff. See
+# https://stackoverflow.com/a/51564689
```

Comment:
In fact this is exactly what Yarn recommends! See "Tip" under https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored.

Reply:
Want me to change the link in the comments?